src/editpage/components/response/MessagesWindow.vue

```diff
@@ -172,6 +172,11 @@ export default {
           // bit strange but allows for no timeout (also for testing)
           this.showingAddError = true;
         }
+        const i = this.messages.length - 1;
+        this.$refs.messageElts[i].scrollIntoView({
+          behavior: 'smooth',
+          block: 'start',
+        });
```

Comment:
And then use that ref here like `this.$refs.addErrorElement.scrollIntoView(...)`
Inside of an request animation frame after both the `this.showingAddError = true` spots

Something like:
```js
this.showTimeout = setTimeout(() => {
  this.showingAddError = true;
  requestAnimationFrame(() => {
    this.$refs.addErrorElement.scrollIntoView({
      behavior: 'smooth',
      block: 'start',
    });
  });
}, this.addErrorTimeout);
```